src/uhi_serialization/hdf5_serialization.py

```diff
@@ -1,328 +1,415 @@
-import h5py
-import boost_histogram as bh
-from pathlib import Path
+from __future__ import annotations
+
 import ast
+from pathlib import Path
+from typing import Any
+
+import boost_histogram as bh
+import h5py
 import numpy as np
 
-CONSTS = {
```

Comment:
Dict of dict is harder to statically type (needs TypedDict), and it wasn't using the outer layer at all anyway.